tensorrt_llm/_torch/modules/fused_moe/fused_moe_wide_ep.py

```
@@ -388,6 +415,10 @@ def forward_chunk(
 
         is_first_call, is_last_call = repeating_info
 
+        print(
+            f"wide_ep forward_chunk: layer_load_balancer={self.layer_load_balancer}, is_first_call={is_first_call}, is_last_call={is_last_call}, x shape: {getattr(x, 'shape', None)}, router_logits shape: {getattr(router_logits, 'shape', None)}, use_all_to_all: {use_all_to_all}, output_dtype: {output_dtype}, all_rank_num_tokens: {all_rank_num_tokens}, all_rank_max_num_tokens: {all_rank_max_num_tokens}, use_dp_padding: {use_dp_padding}, repeating_info: {repeating_info}"
```

Ruff (0.12.2)

Line too long (472 > 120)

(E501)